UI/src/model/locationSearch.js

```diff
@@ -147,19 +161,24 @@
      * @returns {*}
      */
     var  massTransitStopLiviIdSearch = function(input) {
-      return $.when(backend.getMassTransitStopByLiviIdForSearch(input.text), backend.getCoordinatesFromRoadAddress(input.text)).then(function(result,roadData) {
-        var returnObject = roadLocationAPIResultParser(roadData);
-        if (_.get(result[0], 'success')) {
-          var lon = _.get(result[0], 'lon');
-          var lat = _.get(result[0], 'lat');
-          var nationalid=_.get(result[0], 'nationalId');
+      return $.when(backend.getMassTransitStopByLiviIdForSearch(input.text)).then(function(result) {
```

Comment:
Why we delete the code "backend.getCoordinatesFromRoadAddress(input.text)" ?

Reply:
Because the input is only theLivid (something like Livid123456) and VKM require tie, osa, etaisyys value to return values.
Otherwise it will return HTTP 500

Reply:
Ok seems fine, I will resolve this thread.